src/components/ContentNav/Item/Item.storybook.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import centered from '@storybook/addon-centered';
+
+import { Item, ItemProps } from '.';
+import { SubItem } from '../SubItem';
+
+const subItems = [
```

Comment:
This is a mock, but it's in denial. Put it the same '__mocks__' directory as the other ones.